org.eclipse.scout.rt.api/src/main/java/org/eclipse/scout/rt/api/code/CodeResource.java

```diff
@@ -127,11 +132,44 @@ protected void mergeCodeTexts(CodeDo from, Collection<CodeDo> targetList) {
     if (target == null) {
       return;
     }
-    from.getTexts().forEach(target::withText);
+    mergeTexts(from.getTexts(), target.getTexts(), target::withText);
     Collection<CodeDo> targetChildren = getChildCodes(target);
     from.getChildren().forEach(code -> mergeCodeTexts(code, targetChildren));
   }
 
+  /**
+   * Merges the to text maps (languageTag to text). If there is already an entry in the target map with the same text
```

Comment:
```suggestion
   * Merges two text maps (languageTag to text). If there is already an entry in the target map with the same text
```

What exactly does "merge" mean? Is a new map created or is one of the maps altered? It seems that the targetTexts map is directly updated. If so, the description should probably be more like "Merges the entries from the fromTexts into the targetTexts map". But what is the "withText" consumer used for? Why are the maps not just updated? Or a new merged map returned?